it/mail/chrome/messenger/localMsgs.properties

```diff
@@ -99,14 +99,14 @@ pop3PreviouslyEnteredPasswordIsInvalidPrompt=Please enter a new password for use
 # Status - write error occurred
 ## @name POP3_MESSAGE_WRITE_ERROR
 ## @loc None
-4006=Non è possibile scrivere i messaggi nella casella di posta. Assicurarsi di possedere i privilegi in scrittura e sufficiente spazio su disco per ricopiare la casella di posta.
+4006=Non è possibile scrivere i messaggi nella casella e-mail. Assicurati che il file system ti consenta di scrivere i privilegi e assicurati di avere sufficiente spazio su disco per copiare la casella e-mail.
```

Comment:
CORRECTIONS NEEDED.

The two versions are too different to determine which one is correct. Please check again with the source text in English.